app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -355,6 +359,50 @@ class HomeFragment : Fragment() {
             requireComponents.analytics.metrics.track(Event.SearchBarTapped(Event.SearchBarTapped.Source.HOME))
         }
 
+        view.toolbar_wrapper.setOnLongClickListener {
+            val clipboard = view.context.components.clipboardHandler
+            val customView = LayoutInflater.from(view.context)
+                .inflate(R.layout.browser_toolbar_popup_window, null)
+            val popupWindow = PopupWindow(
+                customView,
+                LinearLayout.LayoutParams.WRAP_CONTENT,
+                view.context.resources.getDimensionPixelSize(R.dimen.context_menu_height),
+                true
+            )
+
+            popupWindow.elevation =
+                view.context.resources.getDimension(R.dimen.mozac_browser_menu_elevation)
+
```

Comment:
This code is similar to BrowserToolbarView, except there we have a fix:
`     // This is a workaround for SDK<23 to allow popup dismissal on outside or back button press
            // See: https://github.com/mozilla-mobile/fenix/issues/10027
            popupWindow.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))`

I'm assuming this is needed also here